src/api/app/jobs/consistency_check_job.rb

```diff
@@ -9,53 +8,32 @@ def perform
   end
 
   def check_one_project(project, fix)
-    unless Project.valid_name?(project.name)
-      @errors << "Invalid project name #{project.name}\n"
-      if fix
-        # just remove it, the backend won't accept it anyway
-        project.commit_opts = { no_backend_write: 1 }
-        project.destroy
-      end
-      return
-    end
     @errors << package_existence_consistency_check(project, fix)
     @errors << project_meta_check(project, fix)
+    @errors.flatten.reject(&:empty?).join("\n")
   end
 
   def _perform(fix)
-    @errors = project_existence_consistency_check(fix)
+    @errors << project_existence_consistency_check(fix)
     Project.find_each(batch_size: 100) { |project| check_one_project(project, fix) }
     return if @errors.empty?
-    @errors = "FIXING the following errors:\n" << @errors if fix
-    Rails.logger.error('Detected problems during consistency check')
```

Comment:
The only instrumentation used in production is errbit. Random log without a strategy or reasoning behind is just noise.